### PR TITLE
Viper uat homepage

### PR DESCRIPTION
**Pull request by kanastasakis26**

### CODE REVIEW

File: src/sass/mmui-styles/mmui_home.scss [hunk 1]
```diff
@@ -36,6 +36,7 @@
 
   a{
     color: white;
+    text-decoration: underline;
```
Comment (xyz8983) [inline src/sass/mmui-styles/mmui_home.scss]: huh now the button wants a underline again?

Comment (kanastasakis26) [reply src/sass/mmui-styles/mmui_home.scss]: I made the text-decoration for mmui-link-secondary-button !important. That should resolve the issue.

File: src/sass/mmui-styles/mmui_home.scss [hunk 2]
```diff
@@ -21,6 +21,17 @@
     color: $mm-white;
   }
 
+  a{
+    color: white;
+    text-decoration: underline;
```
[the remark this answers is not in the document]
Comment (kanastasakis26) [reply src/sass/mmui-styles/mmui_home.scss]: I made the text-decoration for mmui-link-secondary-button !important. That should resolve the issue.

File: src/sass/mmui-styles/mmui_home.scss (hunk 1 above)
Comment (besseym) [reply src/sass/mmui-styles/mmui_home.scss]: Is there just a `mmui-btn`?

Comment (besseym) [reply src/sass/mmui-styles/mmui_home.scss]: https://github.com/massmutual/mmui-web/blob/master/src/sass/bootstrap/modified-buttons.scss#L36

**Pull request merged**